geotrek/jstests/test.core.js

```diff
@@ -281,15 +281,15 @@ describe('Topology helper', function() {
 
     it('It should go through extremities, even if paths have opposite ways 2', function(done) {
         var topo = Geotrek.TopologyHelper.buildTopologyFromComputedPath(idToLayer,
-                        __inputData(L.latLng([40, 3]), L.latLng([39, 0]), [7, 6]));
+                        __inputData(L.latLng([40, 10]), L.latLng([39, 0]), [7, 6]));
```

Comment:
Je n'ai pas le temps de regarder en détail mais je me demande en quoi le test était erroné. LatLng([40, 3]) me semble bien positionné sur le tronçon n°7 donc ça devrait fonctionner. Il y a peut être quelque chose qui m'échappe mais j'ai l'impression que ça cloche quelque part et que c'est masqué mais pas corrigé par ce changement. Je peux me tromper mais du coup ça m'intéresse de comprendre en quoi le test était erroné.

Reply:
c'est a cause de la serialization, en gros le probleme c'est que si je laisse 3 il n'arrive pas a trouver la sublinestring, il y a une difference entre la valeur ecrite dans le test ( 40, 3) et ou il sera placé en réalité. LE probleme c'est que le test n'est pas super bien fait, car dans la réalité la valeur pour trouver sur quelle sous ligne il est il utilise le point snappé grace au marker et il n'est pas en dure. Pour le test si je met dans la generation des lignes 40,0000001 je vais trouver la sous ligne si je met 40,  je ne la trouve pas. j'ai comme erreur :  *the string "Could not interpolate LatLng(40, 3) within [object Object]" was thrown, throw an Error :)*